redhat-cop/containers-quickstarts
# Add hostname and set Locale, fixes #311

#### What is this PR About?
`hostname` is no longer part of the base image and needs to be explicitly added.
Also, it is recommended to have the locale set to `UTF-8`

#### How do we test this?
Follow instructions in README for building and deploying. Once deployed, from the `rabbitmq` directory
```
../_test/rabbitmq.sh test rabbitmq
Ensure build is executed...
Waiting for all builds to start...
Waiting for build to complete...
Check to see if build failed
Waiting for for all pods to start...
Ensure 'hostname' package is installed
OK: RPM package 'hostname' installed
Ensure locale is set to UTF-8
OK: Locale is set to 'en_US.UTF-8'
Check RabbitMQ cluster status...
OK: no RabbitMQ cluster alarms
OK: there are 3 running nodes in the cluster
Tests Completed Successfully!
```

cc: @redhat-cop/day-in-the-life
